vdslib/src/vespa/vdslib/distribution/group.cpp

```diff
@@ -39,11 +39,9 @@ Group::Group(uint16_t index, vespalib::stringref name,
 
 Group::~Group()
 {
-    for (std::map<uint16_t, Group*>::iterator it = _subGroups.begin();
-         it != _subGroups.end(); ++it)
-    {
-        delete it->second;
-        it->second = 0;
+    for (auto & subGroup : _subGroups) {
+        delete subGroup.second;
+        subGroup.second = 0;
```

Comment:
Consider `nullptr` instead of literal 0 (or just remove it entirely since it's a destructor anyway...)

Reply:
Fixed++ PTAL